internal/dataplane/parser/translators/ingress_atc.go

```diff
@@ -27,8 +27,11 @@ var (
 )
 
 var (
-	NormalIngressExpressionPriority = 1
-	IngressDefaultBackendPriority   = 0
+	ExactPathMatchPriority                  = 256 * 3
+	PrefixPathMatchPriority                 = 256 * 2
+	ImplementationSpecificPathMatchPriority = 256
```

Comment:
Are those values meaningful in any way? I believe we owe the potential reader an explanation why we chose 256 and its multiples here. WDYT?

Reply:
These values are just chosen in arbitrary to make sure that exact > prefix > implementationSpecific, and also leave some intervals between the priorities.

Reply:
added comments.